src/main/java/games/strategy/engine/framework/startup/mc/IServerStartupRemote.java

```diff
@@ -34,7 +34,7 @@
 
   void changeServerGameTo(final String gameName);
 
-  void changeToLatestAutosave(final SaveGameFileChooser.AUTOSAVE_TYPE typeOfAutosave);
+  void changeToLatestAutosave(final SaveGameFileChooser.AutoSaveType typeOfAutosave);
```

Comment:
Just flagging this out of extreme caution because this is a remote interface...

I don't know how RMI serializes enums, but if the enum name is sent across the wire, this may break compatibility.  Might have to run a network game with an old version that triggers this code path to see if it's an issue.